moysklad/moysklad_class_lib.py

```diff
@@ -0,0 +1,206 @@
+""" В модуле хранится описание классов.
+
+"""
+import requests
+import os, sys
+import logging
+import logging.config
+import logger_config
+import project_settings
+
+sys.path.insert(1, project_settings.PROJECT_PATH)
+
+import moysklad.moysklad_privatedata as ms_pvdata
+import moysklad.moysklad_lib as ms_lib
+import moysklad.moysklad_urls as ms_urls
+
+
+
+class MoySklad(object):
+    """ Класс описывает работу с сервисом МойСклад"""
+    token = ''  # токен для работы с сервисом
+    logger = None
+
+    def __init__(self):
+        try:
+            self.token = ms_lib.get_access_token(ms_pvdata.USER, ms_pvdata.PASSWORD)
+            logging.config.dictConfig(logger_config.LOGGING_CONF)
+            self.logger = logging.getLogger("moysklad")
```

Comment:
Довольно распростанено инициализировать так 
```suggestion
            self.logger = logging.getLogger(__name__)
```